draft-ietf-httpbis-retrofit.md

```diff
@@ -287,20 +311,30 @@ Cookie attributes map to Parameters on the Inner List, with the parameter name b
 
 The Expires attribute is mapped to a Date representation of parsed-cookie-date (see {{Section 5.1.1 of COOKIES}}).
 
-For example, these unstructured fields:
+For example, this Set-Cookie field:
 
 ~~~ http-message
 Set-Cookie: lang=en-US; Expires=Wed, 09 Jun 2021 10:18:14 GMT;
                samesite=Strict; secure
-Cookie: SID=31d4d96e407aad42; lang=en-US
 ~~~
 
-can be mapped into:
+would have a mapped value of:
 
 ~~~ http-message-new
-SF-Set-Cookie: ("lang" "en-US"); expires=@1623233894;
+("lang" "en-US"); expires=@1623233894;
                samesite=Strict; secure
```

Comment:
(A new observation.) This doesn't explain why this mapping was chosen.  It's a bit of an odd choice given that cookies might more naturally fit into the dictionary grammar.  I think that this is because cookie names tend to use non-`key` characters (like uPpErCaSe or leading "_").

Reply:
Yes, that's why. I'm usually wary of documenting motivations, as they quickly make a spec very long.

Reply:
This is a giant ball of speculation anyway. What harm is there in explaining what is not particularly obvious? From the examples this is particularly bad, you might avoid some of that pressure to explain by using a `__Secure-` prefix or even just "LanG=en-US" as your example.

Reply:
Fixed.